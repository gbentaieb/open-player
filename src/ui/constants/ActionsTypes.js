// Config Actions
export const MERGE_CONFIG = 'MERGE_CONFIG';

// Core Actions
export const SET_PLAYER_STATE = 'SET_PLAYER_STATE';
export const REQUEST_PLAY = 'REQUEST_PLAY';
export const SEEK_TO = 'SEEK_TO';
export const SET_PLAYER_TIMES = 'SET_PLAYER_TIMES';
