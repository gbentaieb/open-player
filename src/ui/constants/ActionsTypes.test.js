import * as actiontypes from './ActionsTypes';

describe('ActionTypes > Snapshot', () => {
  test('ActionTypes > Snapshot', () => {
    expect(actiontypes).toMatchSnapshot();
  });
});
