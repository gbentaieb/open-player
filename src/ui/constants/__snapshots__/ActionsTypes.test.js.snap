// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionTypes > Snapshot ActionTypes > Snapshot 1`] = `
Object {
  "MERGE_CONFIG": "MERGE_CONFIG",
  "REQUEST_PLAY": "REQUEST_PLAY",
  "SEEK_TO": "SEEK_TO",
  "SET_FORCED_MUTED": "SET_FORCED_MUTED",
  "SET_FULLSCREEN_CONTAINER": "SET_FULLSCREEN_CONTAINER",
  "SET_IS_ACTIVE": "SET_IS_ACTIVE",
  "SET_IS_FULLSCREEN": "SET_IS_FULLSCREEN",
  "SET_PLAYER_STATE": "SET_PLAYER_STATE",
  "SET_PLAYER_TIMES": "SET_PLAYER_TIMES",
}
`;
